main/java/sorts/Sort.java

```diff
@@ -0,0 +1,11 @@
+package sorts;
+
+import java.util.ArrayList;
+
+public interface Sort {
+	public void sortArrayFirstApproach(double[] list);
```

Comment:
Learn about interfaces. Is public for method names needed.

Reply:
No. Public is the default value.